docs/usage.md

```diff
@@ -76,9 +76,8 @@ The simulation mode has some sample data that helps you get a better understandi
     - Execute **Ingest Known Exploited Vulnerabilities (KEV) CVEs** from `CVE` module List View.
     > **Note**: Please wait till ingestion gets over.
 - Ingest ICS Advisory using any of the below ways:
-    - Manualy execute the playbook **Ingest CISA Advisories** to get `ICS Advisory` or `ICS Medical Advisory`.
-    - Execute **Ingest CISA Advisories** from `ICS Advisories` module List View.
-    > **Note** `Active` schedule **Ingest Latest CISA ICS Advisory** and **Ingest Latest CISA ICS Medical Advisory** as to ingest daily released CISA Advisories.
+    - Manually execute the playbook **Ingest CISA ICS Advisory** or **Ingest CISA ICS Medical Advisory** to get `ICS Advisory` or `ICS Medical Advisory` from `ICS Advisories` module List View.

```

Comment:
Suggestion --> To get `ICS Advisory` or `ICS Medical Advisory`,  manually execute the playbook **Ingest CISA ICS Advisory** or **Ingest CISA ICS Medical Advisory**  from `ICS Advisories` module List View.